### PR TITLE
Add System Level Loads Output Variables

### LINKED ISSUE #7799: Addition of Sensible and Moisture Load Values at System Level (after Zone Multipliers are added)
With Defect #7558, there were some changes to moisture load output to bring it in line with sensible load output.  So, now there is consistency in that all of these loads are on a per zone basis without the application of any zone multipliers.  However, there are some circumstances where the loads with the multipliers might be useful for looking at what is happening at the system level.  So, this should add the possibility that these variables at the system level should be output as well.  In addition, a new input file should be added to the test suite to demonstrate all of these variables.

### Details
Some additional details for this issue (if relevant):
 - Platform: all
 - Version of EnergyPlus: Version 9.2
 - Unmethours link or helpdesk ticket number: none

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added (no defect file--modify the existing test suite file: FurnaceWithDXSystemRHcontrol.idf)
 - [x] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by RKStrand**
Pull request overview
---------------------
 - Fixes #7799 
 - This pull request adds variables as requested by @lgu1234 at the system level.  This is follow up work to a previous bug fix that clarified the zone level variables to be consistent between sensible and latent loads.  Fix includes the ability to add six new variables to an IDF, a unit test for a new subroutine modeled after one that was created for the previous bug, a new CI test that demonstrates the addition, and modified/updated documentation for the new variables.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect/NewFeature
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

**Comment by RKStrand** [comment]
@lgu1234 The previous bug was #7558

File: doc/input-output-reference/src/overview/group-zone-controls-thermostats.tex [hunk 1]
```diff
@@ -254,15 +260,27 @@ \subsubsection{Outputs}\label{outputs-042}
...
+This is the predicted sensible load in W required to meet the current zone thermostat cooling setpoint. A positive value indicates a heating load, a negative value indicates a cooling load. This is calculated and reported from the Predict step in the Zone Predictor-Corrector module. This value is NOT multiplied by zone or group multipliers.
+
+\paragraph{Zone System Predicted Sensible Load to Setpoint Heat Transfer Rate {[}W{]}}\label{zone-predicted-sensible-load-to-setpoint-heat-transfer-rate-w}
+
+This is the predicted sensible load in W required to meet the current zone thermostat setpoint at the system level. A positive value indicates a heating load, a negative value indicates a cooling load. This is calculated and reported from the Predict step in the Zone Predictor-Corrector module. For nearly all equipment types, the Predictor-Corrector evaluates the active heating and/or cooling setpoints, determines if the zone requires heating or cooling or is in the deadband, and then passes this single load to the equipment. This value the Zone Predicted Sensible Load to Setpoint Heat Transfer Rate multiplied by zone or group multipliers.
+
```
Comment (lgu1234) [inline doc/input-output-reference/src/overview/group-zone-controls-thermostats.tex:276]: How about for the last sentence (consistent with the next paragraph):
 This value is the Zone Predicted Sensible Load to Setpoint Heat Transfer Rate multiplied by zone or group multipliers.

**Review by lgu1234 — APPROVED** [review]

**Comment by lgu1234** [comment]
@RKStrand I also checked outputs in the test file and found expected results.

File: doc/input-output-reference/src/overview/group-zone-controls-thermostats.tex (hunk 1 above)
Comment (RKStrand) [reply doc/input-output-reference/src/overview/group-zone-controls-thermostats.tex:276]: @lgu1234 Nice catch!  I've fixed this and made another commit.

**Comment by lgu1234** [comment]
@Myoldmopar There is an unrelated failure in CI Mac test as EnergyPlusFixture.Test_UnitaryHybridAirConditioner_Unittes. Is it OK to merge?

**Comment by rraustad** [comment]
All CI's are reporting diffs in 7 ERR files. Are these coil water flows now non-zero? Has that warning been removed? This branch is behind develop by 3 commits, but not sure if that is causing this difference.

    -   ** Warning ** InitController: Controller:WaterCoil="VAV WITH REHEAT_HEATC_CONTROLLER", Maximum Actuated Flow is zero.

**Comment by RKStrand** [comment]
@lgu1234 @rraustad Not sure if it being behind is the problem or not.  So, I went ahead and merged develop into this branch.  That should launch another CI and hopefully answer that question.  Hoping this resolves whatever the issue was...

**Comment by lgu1234** [comment]
@rraustad @Myoldmopar Differences in ERR are gone. Is it OK to merge?

**Comment by rraustad** [comment]
@Myoldmopar when new report variables are added are they also required to be listed somewhere as are report variables that are changed? Otherwise this looks good.

**Comment by mjwitte** [comment]
@rraustad No, we don't typically list new output variables in the OutputChanges doc. And transition doesn't need to know about them. So just the I/O Ref is enough.

**Comment by RKStrand** [comment]
@lgu1234 Did this end up getting merged or are you still waiting on @Myoldmopar to give you the green light?

**Comment by lgu1234** [comment]
@RKStrand I am waiting for green light from Edwin.

File: testfiles/5ZoneSwimmingPool.idf [hunk 2]
```diff
@@ -2,8 +2,8 @@
 !-Option OriginalOrderTop UseSpecialFormat
 !-NOTE: All comments with '!-' are ignored by the IDFEditor and are generated automatically.
 !-      Use '!' comments if they need to be retained when using the IDFEditor.
-! 5ZoneAirCooled.idf
-! Basic file description:  1 story building divided into 4 exterior and one interior conditioned zones and return plenum.
+! 5SwimmingPool.idf
```
Comment (Myoldmopar) [inline testfiles/5ZoneSwimmingPool.idf:5]: Good rename except it is 5ZoneSwimmingPool.  No need for another commit just to change this though.

File: tst/EnergyPlus/unit/ZoneTempPredictorCorrector.unit.cc [hunk 3]
```diff
@@ -1564,3 +1564,136 @@ TEST_F(EnergyPlusFixture, ReportMoistLoadsZoneMultiplier_Test)
...
+    EXPECT_NEAR(ExpectedResult,SingleZoneHeatRate,AcceptableTolerance);
+    ExpectedResult = 3600.0;
+    EXPECT_NEAR(ExpectedResult,SingleZoneCoolRate,AcceptableTolerance);
+    ExpectedResult = 3600.0;
+    EXPECT_NEAR(TotOutReq,ExpectedResult,AcceptableTolerance);
+    ExpectedResult = 7200.0;
+    EXPECT_NEAR(OutReqToHeatSP,ExpectedResult,AcceptableTolerance);
+    ExpectedResult = 10800.0;
+    EXPECT_NEAR(OutReqToCoolSP,ExpectedResult,AcceptableTolerance);
+}
```
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/ZoneTempPredictorCorrector.unit.cc:1699]: Unit test is good.

**Review by Myoldmopar — APPROVED** [review]
Code changes look just fine.

**Comment by Myoldmopar** [comment]
CI is happy, just the report variable diffs.  This is good to go.  Merging.

**Pull request merged**